Dockerfile

```diff
@@ -1,38 +1,55 @@
-FROM amazoncorretto:11-alpine AS build
+FROM eclipse-temurin:11-jdk-noble AS build
 
 ARG CASSANDRA_STRESS_VERSION
 
+ENV LD_LIBRARY_PATH="/lib/x86_64-linux-gnu:/usr/local/lib:/usr/lib:/lib:/lib64:/usr/local/lib/x86_64-linux-gnu"
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV TZ="UTC"
+
 WORKDIR /app
 
 COPY . .
 
-RUN apk update \
-    && apk upgrade \
-    && apk add apache-ant bash \
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime \
+    && echo "$TZ" > /etc/timezone \
+    && apt update \
+    && apt install -y ant \
     && ant realclean \
     && mkdir -p build lib \
     && ant -Drelease=true -Dversion="$CASSANDRA_STRESS_VERSION" artifacts \
     && bash ./SCYLLA-VERSION-GEN \
     && cp build/SCYLLA-* build/dist/
 
-FROM amazoncorretto:11-alpine AS production
+FROM eclipse-temurin:11-jre-noble AS production
 
 ENV SCYLLA_HOME=/scylla-tools-java
 ENV SCYLLA_CONF=/scylla-tools-java/conf
+ENV PATH=$PATH:$SCYLLA_HOME/tools/bin
+ENV LD_LIBRARY_PATH="/lib/x86_64-linux-gnu:/usr/local/lib:/usr/lib:/lib:/lib64:/usr/local/lib/x86_64-linux-gnu"
+ENV DEBIAN_FRONTEND="noninteractive"
+ENV TZ="UTC"
 
 WORKDIR $SCYLLA_HOME
 
-ENV PATH=$PATH:$SCYLLA_HOME/tools/bin
-
 COPY --from=build /app/build/dist .
 
-RUN apk update \
-    && apk upgrade \
-    && apk add bash \
+RUN ln -snf "/usr/share/zoneinfo/$TZ" /etc/localtime \
+    && echo "$TZ" > /etc/timezone \
+    && apt update \
+    && apt upgrade -y \
+    && apt install -y libsnappy-java libsnappy-jni \
```

Comment:
nit: if that's your final layer (last FROM statement), then you should do the cleaning here as well

Reply:
there is no need, It's done later, this is the same RUN, diff looks a bit weird but when you open the file, everything in inside one RUN